.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+**Description**
+<!-- Please describe what you have changed or added -->
+
+**How has this been tested?**
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+**Screenshots**
+If applicable, add screenshots that verify the behavior of your changes
```

Comment:
```suggestion
If applicable, add screenshots to show the visual change.
```

Reply:
FWIW I copied that sentence from the Gutenberg repo if you wanna suggest that change there as well :-)

Reply:
You seem to have used an older version. https://github.com/WordPress/gutenberg/tree/master/.github